### ISSUE TITLE
WAYoutube - TypeError: 'NoneType' object is not callable

### ISSUE THREAD

**Issue by SweetShyGirl** [danielcardeenas/MacBot#1]
```
  File "/home/user/Bot/app/receiver/receiver.py", line 53, in handle_global_receivers
    receiver.fn(message_entity)
TypeError: 'NoneType' object is not callable


```

I get that error when I did !yt then sends this link
https://youtu.be/J3thQUC_tLY

**Comment by SweetShyGirl** [comment]
I'm using python 2

**Comment by danielcardeenas** [comment]
Fixed on: 52c983b
